advanced/sr_gui_fingertip_visualization/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ssr_gui_fingertip_visualization)
```

Comment:
I think it's sr_gui_fingertip_visualization, right now it doesn't build because it doesn't match with the project name on the xml